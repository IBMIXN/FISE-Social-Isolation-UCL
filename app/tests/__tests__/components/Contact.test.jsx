import React from 'react';
import ReactDOM from 'react-dom';
import Contact from '../../../src/components/Contact';

import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })

it('contact render component unit test', () => {
  const wrapper = shallow(<Contact />)
  
  expect(toJson(wrapper)).toMatchSnapshot();
});

// it('integration test', () => {
//   const wrapper = mount(<Contact />)

//   expect(toJson(wrapper)).toMatchSnapshot();
// });