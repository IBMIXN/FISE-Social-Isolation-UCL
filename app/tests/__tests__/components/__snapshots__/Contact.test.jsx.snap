// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contact render component unit test 1`] = `
<EmotionCssPropInternal
  __EMOTION_LABEL_PLEASE_DO_NOT_USE__="Contact"
  __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
  css={
    Object {
      "map": undefined,
      "name": "haddc6",
      "next": undefined,
      "styles": "
      width: 100%;
      height: 11em;
      clip-path: circle();
    ",
      "toString": [Function],
    }
  }
>
  <EmotionCssPropInternal
    __EMOTION_LABEL_PLEASE_DO_NOT_USE__="Contact"
    __EMOTION_TYPE_PLEASE_DO_NOT_USE__="img"
    alt="p"
    css={
      Object {
        "map": undefined,
        "name": "1njyuqr",
        "next": undefined,
        "styles": "
        width: 100%;
        object-fit: cover;
      ",
        "toString": [Function],
      }
    }
  />
</EmotionCssPropInternal>
`;
